ts/screens/onboarding/NewOnboardingEmailInsertScreen.tsx

```diff
@@ -0,0 +1,287 @@
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "email.insert.help.title",
+  body: "email.insert.help.content"
+};
+
+/**
+ * A screen to allow user to insert an email address.
+ */
+const NewOnboardingEmailInsertScreen = (props: Props) => {
+  const dispatch = useIODispatch();
+
+  // TODO: change this state logic and name (this value will be retrive by the backend)
+  const [isCduEmail] = useState<boolean>(false);
+
+  const viewRef = React.createRef<View>();
+
+  const profile = useIOSelector(profileSelector);
+  const optionEmail = useIOSelector(profileEmailSelector);
+
+  const isLoading = useMemo(
+    () => pot.isUpdating(profile) || pot.isLoading(profile),
+    [profile]
+  );
+
+  const reloadProfile = useCallback(
+    () => dispatch(profileLoadRequest()),
+    [dispatch]
+  );
+
+  const acknowledgeEmailInsert = useCallback(
+    () => dispatch(emailInsert()),
+    [dispatch]
+  );
+  const updateEmail = useCallback(
+    (email: EmailString) =>
+      dispatch(
+        profileUpsert.request({
+          email
+        })
+      ),
+    [dispatch]
+  );
+
+  const [email, setEmail] = useState(isCduEmail ? optionEmail : O.none);
+
+  /** validate email returning three possible values:
+   * - _true_,      if email is valid.
+   * - _false_,     if email has been already changed from the user and it is not
+   * valid.
+   * - _undefined_, if email field is empty. This state is consumed by
+   * LabelledItem Component and it used for style pourposes ONLY.
+   */
+  const isValidEmail = () =>
+    pipe(
+      email,
+      O.map(value => {
+        if (
+          EMPTY_EMAIL === value ||
+          !validator.isEmail(value) ||
+          isSameEmailToChange()
+        ) {
+          return undefined;
+        }
+        return E.isRight(EmailString.decode(value));
+      }),
+      O.toUndefined
+    );
+
+  /**
+   * This function control if the email is the same that the user need to change
+   * @returns boolean
+   */
+  const isSameEmailToChange = () =>
+    !isCduEmail ? areStringsEqual(email, optionEmail, true) : false;
+
+  /**
+   * This function control if the email is already used
+   * @returns boolean
+   *
+   * TODO: this function need to be integrated with API that control if the email already exists
+   */
+  const isExistingEmail = () => {
+    const showAlertExistsEmail: boolean = false;
+    // TODO: insert the selector to control the email
+    if (showAlertExistsEmail) {
+      Alert.alert(
+        I18n.t("email.newinsert.alert.modaltitle"),
+        I18n.t("email.newinsert.alert.modaldescription"),
+        [
+          {
+            text: I18n.t("email.newinsert.alert.modalbutton"),
+            style: "cancel"
+          }
+        ]
+      );
+    }
+    return showAlertExistsEmail;
+  };
+
+  const navigateToEmailReadScreen = useCallback(() => {
+    props.navigation.dispatch(StackActions.popToTop());
+    props.navigation.navigate(ROUTES.ONBOARDING_READ_EMAIL_SCREEN);
```

Comment:
This must be changed. There was a bug: https://github.com/pagopa/io-app/pull/4948/files#diff-cddf0ca756ef05b2f6f42582153d711176307d0390220dc4c89988bfed6755ee

Reply:
done!